spec/features/login_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'User signs in', type: :system do
+  before do
+    @user = create :user
+    visit new_user_session_path
+  end
+
+  scenario 'valid with correct credentials' do
```

Comment:
- [ ] Nice work writing tests for your project, kindly check,  your tests are failing, see the screenshot below.
![error-test](https://user-images.githubusercontent.com/48371156/229289402-d635474f-5642-4578-8339-6eae4db87fab.png)